pkg/api/api.go

```diff
@@ -211,13 +228,32 @@ func (h *APIHandler) Root(w http.ResponseWriter, r *http.Request) {
 
 // License handles /_license get requests
 func (h *APIHandler) License(w http.ResponseWriter, r *http.Request) {
+	h.recordRequest(r, nil)
 	h.metrics.licenseTotalMetrics.Add(context.Background(), 1, metric.WithAttributeSet(requestAttributes(r)))
 	license := fmt.Sprintf("{\"license\" : {\"status\" : \"active\", \"uid\" : \"%s\", \"type\" : \"trial\", \"expiry_date_in_millis\" : %d}}", h.UUID.String(), h.Expire.UnixMilli())
 	w.Header().Set(http.CanonicalHeaderKey("Content-Type"), "application/json")
 	w.Write([]byte(license))
 	return
 }
 
+// History handles /_history get requests
+func (h *APIHandler) History(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set(http.CanonicalHeaderKey("Content-Type"), "application/json")
+	w.WriteHeader(http.StatusOK)
+
+	h.historyMu.Lock()
+	h.historyMu.Unlock()
+	json.NewEncoder(w).Encode(h.history)
+	return
+}
+
+func (h *APIHandler) recordRequest(r *http.Request, body []byte) {
+	log.Printf("%s %s\n%s", r.Method, r.URL.RequestURI(), body)
```

Comment:
can we move the printing of the request out of the api package and move it over to the cmd package (probably behind a verbose flag).  That way if you use the api package in a test you aren't polluting stdout.